server.js

```diff
@@ -18,8 +19,65 @@ import env from "./config/env.generated.json";
 const defaultPort = 3000;
 const PORT = env.PORT || process.env.PORT || defaultPort;
 
+/**
+ * Some app security settings
+ */
+
+var securityHeaders = {
+  directives: {
+    defaultSrc: [
+      `\'none\'`
```

Comment:
`\'` is nothing unless the string itself is quoted with single quotes, so if we're using templating strings there should be no reason whatsoever to escape the single quotes inside of that. Do these need single quotes at all?